src/index.ts

```diff
@@ -1,4 +1,5 @@
 import { getInput, setFailed, warning } from '@actions/core';
+import { info } from '@actions/core/lib/core';
```

Comment:
Please don't import from internals of modules. We should consume only the advertised API.

Reply:
fixed with: https://github.com/ridedott/dependabot-auto-merge-action/pull/233/commits/8fa422481dda410cd7bc52b6c04ead5cbcab9c54